### DISCUSSION TITLE
Import Google Photos metadata

### DISCUSSION THREAD

**Discussion by axewater** [photoprism/photoprism#865] [Q&A] [answered]
I have exported files from Google Photos, it includes metadata files. Would it be possible to import this into photoprism somehow ?

**Comment by lastzero** [comment]
Metadata in JSON sidecar files will be indexed by PhotoPrism, Google albums not yet from what I remember. They had really bad quality in our tests, many automatically created by Google.

**Comment by lastzero** [comment, marked as answer]
Docs have been updated: https://docs.photoprism.org/user-guide/use-cases/google/
